### PR TITLE
Fix: @amount should be optional

### PR DESCRIPTION
**Pull request by renaatdemuynck**
The Less.js docs say the following:
"weight: Optional, a percentage balance point between color and black, defaults to 50%."

### CODE REVIEW

File: lib/Less/Functions.php [hunk 1]
```diff
@@ -813,11 +813,11 @@ private function _isa($n, $type) {
 		return is_a($n, $type) ? new Less_Tree_Keyword('true') : new Less_Tree_Keyword('false');
 	}
 
-	public function tint($color, $amount) {
+	public function tint($color, $amount = null) {
```
Comment (PhrozenByte) [inline lib/Less/Functions.php]: Even though the default value of `$amount` of `Less_Functions::mix()` is also 50, you shouldn't leave `Less_Functions::tint()`'s default `$amount` up to `Less_Functions::mix()`. Just think that less.js changes the default `$amout` of `Less_Functions::mix()` to e.g. 66.67 - it would cause an unexpected side-effect.

Comment (renaatdemuynck) [inline lib/Less/Functions.php]: @PhrozenByte You're absolutely right. I shouldn't have made that assumption. I will fix this.

Comment (seven-phases-max) [inline lib/Less/Functions.php]: Actually the assumption was fine. In fact that "50%" in `less.js` implementation is not something really intended in `tint`/`shade` (in fact neither default value makes any fair sense there) but a direct consequence of the author of those functions just did not specify the argument (probably for the sake of less verbosity) and therefore [`undefined` amount](https://github.com/less/less.js/blob/master/lib/less/functions/color.js#L312) parameter in `tint`/`shade` explicitly maps to `50%` amount in `mix` (where it makes perfect sense).

In other words, the documentation only reflects simplified/weird implementation rather than really something reasonable. (So `$amount = null` would reflect exactly what they do in `less,js`).

Comment (PhrozenByte) [inline lib/Less/Functions.php]: Oh, in fact, they're doing it upstream wrong as well. My fault, sorry.

Comment (renaatdemuynck) [inline lib/Less/Functions.php]: Ok, so since this is a port of `less.js`, we should do it as they did. I removed my second commit and will create a new pull request.
